Project/Assets/Prairie/Framework/Script/Twine/TwineNode.cs

```diff
@@ -73,149 +79,157 @@ private void Start()
-            //                visibleNodeIndex = 0;
-            //            }
-            //            print(visibleNodeIndex);
-            //            print(TwineNodeList.IndexOf(this));
-            if (!TwineNodeList.Contains(this))
-            {
-                TwineNodeList.Add(this);
-                //                storyTitle = this.name;
-                //                print(this.name);
-                //                insertIndex = TwineNodeList.IndexOf(this);
-                //                print("printing");
-                //                print(TwineNodeList.Count);
-                //                    foreach (TwineNode item in TwineNodeList) {
-                //                        print(item.name);
-                //                    }
-                //                    if (TwineNodeList.Count > 1){
-                //                        this.isMinimized = true;
-                //                    }
-            }
-            if (Input.GetKeyDown(KeyCode.Tab) && TwineNodeList.IndexOf(this) == 0)
-            {
-                if (visibleNodeIndex == TwineNodeList.Count - 1)
-                {
-                    visibleNodeIndex = 0;
+        if (this.children.Length == 0) {
+            yield return new WaitForSeconds(5);
+            this.Deactivate();
+        }
+    }
+    
+    /// <summary>
+    /// Check if TwineNode has no children. If it
+    ///     doesn't, wait 5 seconds and deactivate.
+    /// </summary>
+    void Update ()
+    {
+            UpdateConditionalLinks();
+            if (this.enabled) {
+                if (Input.GetKeyDown (KeyCode.C) && TwineNodeList.IndexOf(this) == 0 && allMinimized == false){
+                    fanfold = !fanfold;
                 }
-                else
-                {
-                    visibleNodeIndex++;
+                if (Input.GetKeyDown (KeyCode.M) && TwineNodeList.IndexOf(this) == 0){
+                    allMinimized = !allMinimized;
+                }
+                if (!TwineNodeList.Contains(this)){
+                    TwineNodeList.Add(this);
+                }
+                if (Input.GetKeyDown (KeyCode.Tab) && TwineNodeList.IndexOf(this) == 0 && allMinimized == false && fanfold == false){
+                    if (visibleNodeIndex == TwineNodeList.Count - 1) {
+                        visibleNodeIndex = 0;
+                    }
+                    else {
+                        visibleNodeIndex++;
+                    }
+                }
+                if (TwineNodeList.IndexOf(this) == visibleNodeIndex) {
+                    this.isMinimized = false;
+                }
+                else {
+                    this.isMinimized = true;
+                }
+                if (this.isDecisionNode) {
+                    this.isOptionsGuiOpen = true;
+                } else if (this.isConditionNode) {
+                    // get the $color value from global list
```

Comment:
Modify this comment. See review from the last PL.

Reply:
We should probably take out all of the else if for this.isConditionNode.  That's the start of some code for immediate transition nodes, or nodes that automatically activate their first child if its twine variable condition is met.  We've decided not to implement that this year, and we shouldn't have this unused code floating around.  It's not a Teddy thing, though, that's on Lucy and I.